### PR TITLE
Userdata

### PR DESCRIPTION
**Pull request by ravi-kishan**
First steps for issues #148 

Changes: Profile page to view user account created.

### CODE REVIEW

File: app/src/main/java/com/iitp/njack/iitp_connect/data/user/User.java [hunk 1]
```diff
@@ -33,4 +33,18 @@
     String email;
     String firstName;
     String lastName;
+    String roll;
+
+    public User(
```
Comment (dr0pdb) [inline app/src/main/java/com/iitp/njack/iitp_connect/data/user/User.java]: This is redundant. You don't need to add the constructor. The `@AllArgsConstructor` annotation does it for you.

**Review by dr0pdb — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/iitp/njack/iitp_connect/data/user/User.java (hunk 1 above)
Comment (ravi-kishan) [reply app/src/main/java/com/iitp/njack/iitp_connect/data/user/User.java]: The redundant constructor has been removed.

**Review by dr0pdb — APPROVED** [review]
Looks good!

**Comment by dr0pdb** [comment]
@BVShe001 Review it please.

**Comment by sheetal-g9** [comment]
Everything running fine ,profile UI added .

**Pull request merged**